source/common/http/conn_manager_impl.h

```diff
@@ -508,16 +523,19 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       bool saw_connection_close_ : 1;
     };
 
+    // Possibly increases buffer_limit_ to the value of limit.
+    void setBufferLimit(uint32_t limit);
+
     ConnectionManagerImpl& connection_manager_;
     Router::ConfigConstSharedPtr snapped_route_config_;
     Tracing::SpanPtr active_span_{new Tracing::NullSpan()};
     const uint64_t stream_id_;
     StreamEncoder* response_encoder_{};
     HeaderMapPtr response_headers_;
-    Buffer::InstancePtr buffered_response_data_; // TODO(mattklein123): buffer data stat
+    Buffer::WatermarkBufferPtr buffered_response_data_; // TODO(mattklein123): buffer data stat
```

Comment:
I think at this point we can kill these TODOs IMO. (The TODO was here from a long time ago when we didn't have flow control and I was worried).